packages/integration/src/processVanillaFile.ts

```diff
@@ -110,11 +110,7 @@ export async function processVanillaFile({
       cssObjs: fileScopeCss,
     }).join('\n');
 
-    const fileName = `${
-      fileScope.packageName
-        ? `${fileScope.packageName}/${fileScope.filePath}`
-        : fileScope.filePath
-    }.vanilla.css`;
+    const fileName = `${fileScope.filePath}.vanilla.css`;
```

Comment:
Since you removed the package name (which I think is a good change), I think we need to update the rollup docs and tests in regards to preserveModulesRoot. I'll check that.